packages/web/src/components/collection/desktop/CollectionHeader.tsx

```diff
@@ -125,91 +171,102 @@ export const CollectionHeader = (props: CollectionHeaderProps) => {
-          icon={icon}
-          imageOverride={imageOverride}
-          isOwner={isOwner}
-        />
-        <div className={styles.infoSection}>
-          <span className={cn(styles.typeLabel, fadeIn)}>
-            {!isPublished ? (
-              <IconVisibilityHidden className={styles.labelIcon} />
-            ) : null}
-            <p className={styles.label}>
-              {type === 'playlist' && !isPublished ? 'hidden playlist' : type}
-            </p>
-          </span>
-          <TitleComponent
-            className={cn(styles.title, { [styles.editableTitle]: isOwner })}
-            onClick={isOwner ? handleClickEditTitle : undefined}
-          >
-            <Text
-              variant='heading'
-              size='xl'
-              className={cn(styles.titleHeader, fadeIn)}
-            >
-              {title}
-            </Text>
-            <ClientOnly>
-              {isOwner ? <IconPencil className={styles.editIcon} /> : null}
-            </ClientOnly>
-            {isLoading ? <Skeleton className={styles.skeleton} /> : null}
-          </TitleComponent>
-          <Flex>
-            {artistName ? (
+    <Flex direction='column'>
+      {/* Top Section */}
+      <Flex gap='xl' p='l' backgroundColor='white'>
+        {coverArtSizes ? (
+          <Artwork
+            collectionId={collectionId}
+            coverArtSizes={coverArtSizes}
+            callback={handleLoadArtwork}
+            gradient={gradient}
+            icon={icon}
+            imageOverride={imageOverride}
+            isOwner={isOwner}
+          />
+        ) : null}
+        <Flex direction='column' justifyContent='space-between'>
+          <Flex direction='column' gap='xl'>
+            <Flex className={cn(fadeIn)} gap='s' mt='s'>
+              {!isPublished ? <IconVisibilityHidden /> : null}
+              {isPremium ? <IconCart size='s' color='subdued' /> : null}
               <Text
-                variant='title'
-                strength='weak'
-                tag='h2'
-                className={cn(fadeIn)}
+                variant='label'
+                color='subdued'
+                css={{ letterSpacing: '2px' }}
               >
-                <Text color='subdued'>By </Text>
-                <UserLink userId={userId} popover />
+                {isPremium ? 'premium ' : ''}
+                {type === 'playlist' && !isPublished ? 'hidden playlist' : type}
               </Text>
-            ) : null}
+            </Flex>
+            <Flex direction='column' gap='s'>
+              <TitleComponent
+                className={cn(styles.title, {
+                  [styles.editableTitle]: isOwner
+                })}
+                css={{
+                  outline: 0,
+                  border: 0,
+                  background: 'none',
+                  display: 'flex',
+                  alignItems: 'center',
+                  gap: spacing.s,
+                  padding: 0
+                }}
```

Comment:
I think we can do `<Flex as={TitleComponent} gap='s' onClick={isOwner ? ... >` and have it take the props for both

Reply:
Great idea, much nicer approach